### DISCUSSION TITLE
What's the correct validate config for a checkbox group?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#10984] [General]
Hi 

I've used `php artisan vendor:publish --tag=statamic-forms`

Running into a couple of issues with validation

This is my fieldset config for the form

```
type: checkboxes
      display: 'Pick an Option'
      validate:
        - required
        - 'min:1'
        - array
      inline: true
  ```
  
What's the correct validate config for a checkbox group? 

This is the checkboxes partial

```html
<input type="hidden" name="{{ handle }}[]">
{{ foreach:options as="option|label" }}
    <label class="{{ view:checkbox_label }}">
        <input
            class="{{ view:checkbox_input }}"
            type="checkbox"
            name="{{ handle }}[]"
            value="{{ option }}"
            {{ if js_driver }}{{ js_attributes }}{{ /if }}
            {{ if value|in_array:option }}checked{{ /if }}
            {{ if validate|contains:required }}required{{ /if }} // I don't want to use this
        >
        {{ label !== null ? label : option }}
    </label>
    {{ unless inline }}
    <br>
{{ /unless }}
{{ /foreach:options }}
```

`if validate|contains:required` uses browser validation on every checkbox in a group. I'd rather not use browser validation as this requires **every** checkbox to be checked in order for the form to submit. This is only useful for a single checkbox.

https://github.com/user-attachments/assets/21f77e28-11f0-47ae-bafe-b17799fbe353

When I remove ` {{ if validate|contains:required }}required{{ /if }}` browser validation is removed, but the Statamic validation doesn't work?

https://github.com/user-attachments/assets/6092b9fa-8339-4ca3-a453-e7152e94dbee

I can only get the Statamic validation to work if I remove `<input type="hidden" name="{{ handle }}[]">`

https://github.com/user-attachments/assets/a06a53df-8e4c-47bc-8d2b-01635a5b3f66

I'd like to use the correct code as output by `php artisan vendor:publish --tag=statamic-forms` I think this is an issue with the validation config I've used. Any tips would be great!

**Comment by AtmoFX** [comment]
FYI, I have encountered the same issue and created the corresponding https://github.com/statamic/cms/issues/11062
